api/operator.go

```diff
@@ -140,6 +140,8 @@ type SchedulerSetConfigurationResponse struct {
 // PreemptionConfig specifies whether preemption is enabled based on scheduler type
 type PreemptionConfig struct {
 	SystemSchedulerEnabled bool
+	BatchEnabled           bool
```

Comment:
Should these be `Batch/ServiceSchedulerEnabled` so as to match the SystemScheduledEnabled option?